spec/unit/resource/api200/version_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'spec_helper'
+
+RSpec.describe OneviewSDK::Version do
+  include_context 'shared context'
+  describe '#get_version' do
+    it 'calls the uri' do
+      allow(@client_200).to receive(:rest_get).with(described_class::BASE_URI, {})
```

Comment:
This test is not testing the call of method `rest_get`, because the `allow` just permit you call some method, but it is not guaranteed to be called. So, maybe is a better do this: 
```ruby
    expect(@client_200).to receive(:rest_get).with(described_class::BASE_URI, {})
    described_class.get_version(@client_200)
```
because you are expecting that the calling to `described_class.get_version` execute `@client_200.rest_get`

Reply:
Thanks Ricardo for making me understand the difference between allow and expect. I've tried with the code suggested above but when i ran the test, i got the below error: 

OneviewSDK::API200::Version#get_version calls the uri
     Failure/Error: response = client.rest_get(BASE_URI)
expected: ("/rest/version", {})
              got: ("/rest/version")
      # ./lib/oneview-sdk/resource/api200/version.rb:22:in `get_version'
     # ./spec/unit/resource/api200/version_spec.rb:8:in `block (3 levels) in <top (required)>'

After going through the examples, I am able to fix this with the below code:
expect(@client_200).to receive(:rest_get).with(described_class::BASE_URI).and_return(FakeResponse.new(members: []))
described_class.get_version(@client_200)

Please let me know if this is correct.

Reply:
It is correct, because the [`get_version`](https://github.com/HewlettPackard/oneview-sdk-ruby/pull/298/files#diff-d63c2adb31f1c992e768145b198c25a4R22) call is being did just with BASE_URI parameter.
Very good! =)